src/main/scala/beam/agentsim/agents/modalbehaviors/DrivesVehicle.scala

```diff
@@ -840,18 +838,7 @@ trait DrivesVehicle[T <: DrivingData] extends BeamAgent[T] with Stash {
         secondaryFuelLevel = Some(vehicle.secondaryFuelLevelInJoules)
       )
     )
-    schedulerMessage.foreach(message => {
-      val (sessionDuration, energyDelivered): (Long, Double) = vehicle.refuelingSessionDurationAndEnergyInJoules()
-      val chargingEndTick = currentTick + sessionDuration.toInt
-      log.debug(
-        "scheduling EndRefuelSessionTrigger at {} with {} J to vehicle {} to be delivered",
-        chargingEndTick,
-        energyDelivered,
-        vehicle.id
-      )
-      scheduler ! message(EndRefuelData(chargingEndTick, energyDelivered))
```

Comment:
I must be missing something, because as far as I can tell there is no end recharging event that will trigger currently? Aren't non CAV vehicles going to end up stuck?